neurom/apps/morph_stats.py

```diff
@@ -76,9 +74,11 @@ def extract_dataframe(morphs, config, n_workers=1):
         config (dict): configuration dict. The keys are:
             - neurite_type: a list of neurite types for which features are extracted
               If not provided, all neurite_type will be used
-            - neurite: a dictionary {{neurite_feature: mode}} where:
-                - neurite_feature is a string from NEURITEFEATURES or NEURONFEATURES
-                - mode is an aggregation operation provided as a string such as:
+            - neurite:
+                Either a list of features: [feature_name, {kwargs: {}, modes: []}] or
+                a dictionary of features {feature_name: {kwargs: {}, modes: []}}.
+                - kwargs is an optional entry allowing to pass kwargs to the feature function
+                - modes is an aggregation operation provided as a string such as:
```

Comment:
Do we want to keep both formats? Shouldn't we raise a deprecation warning for the dict format?

Reply:
Not sure where we stand wrt to deprecating the old formats. @mgeplf , should I add warnings for all previous config layouts?

Reply:
Hmm, the way I was thinking about adding this was to make, the `kwargs` (which was poorly named), a list:
```
neurite:
    partition_asymmetry:
        kwargs:
           - {}
           - { variant: 'length',  method: 'petilla' }
```
and then `isinstance` kwargs to see if it was a dict (and then use the old behavior), or a list(and then use the new behavior.  I think the config could be more minimal this way; however, the modes would be the same for both `kwargs` variants.

Reply:
I like this. I will change it accordingly!

Reply:
I have implemented @mgeplf suggestion for multiple kwargs. @lidakanari , @adrien-berchet could you please try it and let me know if it works for you?Thanks.

Reply:
LGTM; but let's wait for the feedback on usage.